metrics/access-logs/sniped/go.mod

```diff
@@ -0,0 +1,8 @@
+module sniped
```

Comment:
I don't think this should be in `metrics/access-logs`

It doesn't do anything with metrics (read or publish), and it definitely doesn't do anything with access logs

I guess we lack a "useful miscellaneous tooling" directory.  I would either use `audit/` as the home for tools that help audit our project, or make a new `tools/` directory.  Then put `sniped` under that